### ISSUE TITLE
Improve and expand Alt-text detection

### ISSUE THREAD

**Issue by bitfl0wer** [bitfl0wer/altminder#8]
The following features should be implemented to make the bot a bit smarter (courtesy: @KreerC):

- If the message attached to the image contains alt text in the form "Alt: ...", no reminder should be sent.
- If someone adds an alt text message after the fact, the alt text reminder message should go away on its' own. I don't really know what this behaviour should look like, but here are a few ideas:

1. If you reply to the original message using the proper reply feature AND the message contains "Alt: ...", the alt text reminder message should go away on its' own
2. If you just send a new message that contains the chars "Alt: ...", the alt text reminder message should go away on its' own.
3. We don't implement this second feature at all. I personally think that it makes more sense to include the alt text with the images that get posted and not somewhere later in the chat. Including it as a message instead of the image alt attribute might be fine, but the image and its' descriptions should - in my opinion  - always be together. What do you say, @KreerC ?
